test/e2e/kubernetes/windows/windows.go

```diff
@@ -52,7 +52,7 @@ func (i *WindowsImage) GetImage(version OSVersion) string {
 
 func GetWindowsImage(testImage WindowsTestImages, version OSVersion) string {
 	httpd := WindowsImage{
-		BaseImage: "k8sprow.azurecr.io/kubernetes-e2e-test-images/httpd",
+		BaseImage: "registry.k8s.io/e2e-test-images/httpd",
 		Tags: map[OSVersion]string{
 			LTSC2019: "2.4.39-alpine",
```

Comment:
we could bump this version while we are here: https://github.com/kubernetes/kubernetes/blob/fcfe5dfc2177320833caa451b401f2d1dc5668e7/test/utils/image/manifest.go#L252

Reply:
```
docker pull registry.k8s.io/e2e-test-images/httpd:2.4.39-4
....
Digest: sha256:3fe7acf013d1264ffded116b80a73dc129a449b0fccdb8d21af8279f2233f36e                                         Status: Downloaded newer image for registry.k8s.io/e2e-test-images/httpd:2.4.39-4                                       registry.k8s.io/e2e-test-images/httpd:2.4.39-4         
```

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/3271